src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -642,6 +649,28 @@ private static bool GetEnvironmentVariableAsBool(string name, bool defaultValue)
             return defaultValue;
         }
 
+        /// <summary>
+        /// Send module load telemetry as a metric.
+        /// For modules we send the module name (if allowed), and the version.
+        /// Some modules (CIM) will continue use the string alternative method.
```

Comment:
I am unsure what CIM is here

Reply:
discussed here: https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/?view=powershell-7.2